go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/expectedsh/go-sonic
+module github.com/Jonarod/go-sonic
```

Comment:
@alexisvisco @Jonarod 
I'm sorry, but when I run
`go get -u "github.com/expectedsh/go-sonic/sonic"`
I receive
> module declares its path as: github.com/Jonarod/go-sonic
>                 but was required as: github.com/expectedsh/go-sonic

And I guess, this file merge is the reason. Please revert it back.

Reply:
Ho well ..